pageserver/src/bin/pageserver.rs

```diff
@@ -321,6 +330,12 @@ fn main() -> Result<()> {
                 .takes_value(true)
                 .help("Amount of memory reserved for buffering incoming WAL"),
         )
+        .arg(
+            Arg::with_name("max_file_descriptors")
+                .long("max_file_descriptors")
+                .takes_value(true)
+                .help("Max number of file descriptors to keep open for files"),
```

Comment:
Does postgres do something for other "file descriptors" like sockets/stdio?
Might be good to make the scope of the virtual files more clear in `help`